src/ride_build.cpp

```diff
@@ -168,67 +175,96 @@ void RideBuildWindow::OnClick(WidgetNumber wid_num, const Point16 &pos)
 }
 
 /**
- * Checks whether the air space above the ground at the given location is suited to place a shop of the given height.
+ * Checks whether the air space above the ground at the given location is suited to place a fixed ride of the given height.
  * @param position Coordinate of the base voxel.
  * @param height Height of the object.
- * @return The space is suited to build the shop.
+ * @return The space is suited to build the ride.
  */
 static bool CheckSufficientVerticalSpace(const XYZPoint16& position, const int8 height)
 {
 	for (int8 h = 0; h <= height; ++h) {
```

Comment:
Reading this again, It looks wrong now.

Values `height`, `width`, and `length` are always exclusive when counting from `0` (`height==1` means 1 voxel only, that is, only voxel number `0`). If you meant the inclusive upper bound here, use `last` or `highest`.

I see you're using pre-incrementing. I always use post-incrementing as I find that more natural to read.
Likely in the past it had performance consequences, but nowadays, rewriting this to the best form is a job that the compiler does better than us.
(To be clear, I am not asking for a change.)